src/workflows/rna/rna_singlesample/main.nf

```diff
@@ -160,20 +160,35 @@ workflow run_wf {
           obs_filter += ["filter_ribosomal"]
         }
         stateMapping += ["obs_filter": obs_filter]
+        // If scrublet is skipped, the output should be set to the workflow output
+        if (state.skip_scrublet_filtering) {
+          stateMapping += ["output": state.workflow_output]
+        }
```

Comment:
I don't think this is correct. `do_filter` applies to count filtering and has nothing to do with doublet detection using scrublet?